code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -1,3 +1,5 @@
+#define PROCESS_TIME 6
```

Comment:
6 of what? Do please 0.6 SECONDS

Reply:
6 seconds, though it is the default value for a variable, so maybe PROCESS_TIME_SECONDS?